### PR TITLE
Final group capstone: Detail page

### LINKED ISSUE #5: [6pt] Create `item details page`
- When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description (skip the "Rotate image" button).

  - In the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).
Details

### PR DESCRIPTION
**Pull request by wale-prog**
In this pull request, I have implemented the following: 
- Created the room detail controller and add a show action to respond to requests.
- Created a route for the show endpoint

### CODE REVIEW

File: app/controllers/api/v1/rooms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Api::V1::RoomsController < ApplicationController
+  # before_action :authenticate_user!
+
```
Comment (sntakirutimana72) [inline app/controllers/api/v1/rooms_controller.rb:3]: - Kindly consider enabling authentication on the actions in this controller scope as to access them, users must sign in first.

**Review by sntakirutimana72 — APPROVED** [review]
Hi @wale-prog,

# Status: APPROVED!!!  :champagne: 🥂

Bravo👏!!! You have performed wonderfully.👍

## Highlight! :green_circle: 

:heavy_check_mark:  All expected behaviors are present.
:heavy_check_mark:  Good commit messages
:heavy_check_mark:  No linter errors.

## [Optional] suggestions

- Kindly, see the below comments.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

-----

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @sntakirutimana72.

**Review by Chrissiku — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @wale-prog 👋

Your project is complete! ✔️ 🏁

 Your code is very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ No linter Errors

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aliatayee** [comment]
Hi @wale-prog 

# Status: APPROVED!!! 

You did a great job so far! time to merge.

## Highlight! :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  No linter errors.

Cheers and Happy coding!👏👏👏

**Review by aliatayee — APPROVED** [review]

**Review by safar1212 — APPROVED** [review]
# APPROVED!!!! 🎉
## Hey @wale-prog    😊 😃 

#### Great Job 😄 👏 You've done it. Nice completing this milestone 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**